lib/Service/SettingsService.php

```diff
@@ -240,6 +243,10 @@ public function setMinimumConfidence($confidence) {
 	 * The next settings are advanced preferences that are not available in gui.
 	 * See: https://github.com/matiasdelellis/facerecognition/wiki/Settings#hidden-settings
 	 */
+	public function getModelPath(): string {
+		return $this->config->getSystemValue(self::SYSTEM_MODE_PATH, null);
+	}
+	
```

Comment:
Move it here: https://github.com/matiasdelellis/facerecognition/pull/390/files#diff-0d0e5b28c99581ae724962d51121ee75bb2552319070923c2aa9112dab4f2e74R303

..with the appropriate comment. :wink:

Reply:
I meant to just change the getModelfolder function here. Not all new code.. This doesn't even work, because you don't have rootFolder, etc.